buildenv/jenkins/common/build.groovy

```diff
@@ -300,6 +300,19 @@ def archive_sdk() {
         def testDir = "test"
 
         dir(OPENJDK_CLONE_DIR) {
+            // Code coverage files archive for report
+            if (params.CODE_COVERAGE) {
+                // Use WORKSPACE since code coverage gcno files are in vm folder, java is in jdk and images folder, and omr and openj9 folders are in workspace.
+                def codeCoverageDir = "${env.WORKSPACE}"
+                if (SPEC.contains('zos')) {
+                    sh "touch ${CODE_COVERAGE_FILENAME}"
```

Comment:
This `touch` command is common: it should be moved before the `if` statement.

Reply:
The `touch` command has been moved before `if`. Thanks for pointing out.